kamon-datadog/src/main/resources/reference.conf

```diff
@@ -37,6 +37,16 @@ kamon {
     # this pattern:
     #    application.entity-name.metric-name
     application-name = "kamon"
+
+    # All time values are collected in nanoseconds,
+    # to scale before sending to datadog set "time-units" to "s" or "ms" or "µs".
+    # Value "ns" is equivalent to omitting the setting
+    # time-units = "ns"
```

Comment:
if scaling is not needed (values are commented), then there's one actor less in the chain and scaling from 'ns' to 'ns' is omitted. I see this as a nice feature, but it does add more executable code, so i thought it's good if people decide on their own if they want it or not. If they want - they set proper values, not defaults that do nothing.
does it make sense?